typing_extensions/src/typing_extensions.py

```diff
@@ -1377,8 +1381,11 @@ def __getitem__(self, params):
                                 "with at least two arguments (a type and an "
                                 "annotation).")
             else:
-                msg = "Annotated[t, ...]: t must be a type."
-                tp = typing._type_check(params[0], msg)
+                if type(params[0]).__name__ == "_ClassVar":
```

Comment:
This should also check that params[0] is a `typing._TypingBase`.

Reply:
done, thanks!